### DISCUSSION TITLE
Update user email failed

### DISCUSSION THREAD

**Discussion by Zehir** [pocketbase/pocketbase#2938] [Q&A] [answered]
Hello, I am trying to update the user email but each time I get an error;

```javascript
await client.collection('user').update(result.record.id, {
    email: 'foo@bar.fr',
})
```

The http client send the payload :
PATCH http://127.0.0.1:8090/api/collections/user/records/nv4aqqrokjtcdeg 
```json
{"email":"foo@bar.fr"}
```
I get this result from the api ;
```json
{
  "code": 400,
  "message": "Something went wrong while processing your request.",
  "data": {
    "email": {
      "code": "validation_in_invalid",
      "message": "Must be a valid value."
    }
  }
}
```

If I put the same value from the admin UI it's get updated. I don't know what wrong

**Comment by ganigeorgiev** [comment, marked as answer]
The `email` field can be changed directly only by admins or users with "Manage API rule" access.

In case regular users want to change their email, they'll have to send `requestEmailChange()` request.

**Reply by Zehir** [reply]
Thanks for your réponse, I will do that
